documentation/commands.md

```diff
@@ -1,66 +1,26 @@
 # Commands
 
-Commands in the form of JSON messages are used to start and stop file writing.
+Commands are in the form of flatbuffers as defined in [this repository](https://github.com/ess-dmsc/streaming-data-types).
+We strongly recommend that you do not use this repo directly but instead make use of the [*file-writer-control*](https://github.com/ess-dmsc/file-writer-control) Python library.
 
 Commands are sent through Kafka via the broker and topic specified by the
-`--command-uri` option.
-
-Note: some example commands can be found in the system tests.
-
-## Command to start writing
```

Comment:
@matthew-d-jones seeing as you are the one who implemented the use of flatbuffers for start and stop messages, would you mind writing some documentation on that? 😀 Alternatively, I can roll back this change and we can continue to ignore the fact that the documentation has been out of date for the past 15 months. 😁

Reply:
Ticket is ECDC-1897. I need to make some progress on some scipp work, so I can't pick it up for a few days. The docs don't really need deleting, it is all basically the same except that they are fields in a flatbuffer rather than JSON. For now you could revert the change, remove mention of JSON, and add a note that the docs are being updated soon?

Reply:
The way the pool works and needs to be set up does need describing. The tests pass and I trust it works for you, but that's not the point of a review.

Reply:
The removal of outdated documentation has been reverted.